src/Paket.Core/Paket.Core.fsproj

```diff
@@ -60,17 +60,17 @@
   -->
   <Import Project="$(SolutionDir)\.paket\paket.targets" />
   <ItemGroup>
+    <Compile Include="..\..\paket-files\fsprojects\FSharp.TypeProviders.SDK\src\AssemblyReader.fs">
+      <Paket>True</Paket>
+      <Link>Common//AssemblyReader.fs</Link>
+    </Compile>
     <None Include="paket.references" />
     <EmbeddedResource Include="embedded\Paket.Restore.targets" />
     <Compile Include="Common\AssemblyInfo.fs" />
     <Compile Include="Common\CustomAssemblyInfo.fs" />
-    <Compile Include="..\..\paket-files\fsprojects\FSharp.TypeProviders.StarterPack\src\AssemblyReader.fs">
-      <Paket>True</Paket>
-      <Link>Common/AssemblyReader.fs</Link>
-    </Compile>
     <Compile Include="..\..\paket-files\fsharp\FAKE\src\app\FakeLib\Globbing\Globbing.fs">
       <Paket>True</Paket>
-      <Link>Common/Globbing.fs</Link>
+      <Link>Common//Globbing.fs</Link>
```

Comment:
why has it been added as `Common//Globbing.fs` ?

Reply:
can you revert that? doesn't look good

Reply:
I can revert it. Weird that it was added.

Reply:
I'm using Mac OS, vs code, et.c. to edit the files. Could that be why?